data-products-dp0-3/heliolinc3d-tracklet-linking-orbit-fitting.rst

```diff
@@ -0,0 +1,71 @@
+The algorithm identifies and links together little sequences of typically 6-20 sources that could comprise
+repeated detections of a new asteroid moving in its orbit around the Sun. These sets of
+detections (called 'linkages') are formed in two stages. First, ‘tracklets’ of observations
+are identified, where a tracklet comprises at least two images within a single night. Next,
+tracklets from multiple nights are linked together. LSST specifications state that a valid
+linkage must include at least three tracklets, each from a different night, and all within
+a 14-day period. Each linkage meeting these criteria constitutes a candidate asteroid discovery.
+After the full set of candidate linkages has been produced, they are culled and refined through
+orbit fitting and other analyses. The final product is a purified set of thousands of
+non-overlapping linkages, each of which has an orbit-fit with sub-arcsecond astrometric residuals.
+These linkages -- each comprising a probable new asteroid discovery -- are submitted to the
+`Minor Planet Center <https://minorplanetcenter.net>`_ (MPC) for confirmation and publication.
+The tracklet linking and orbit fitting procedure is illustrated in the infographic provided above.
+
+During operations, the tracklet linking and orbit fitting process will consist of the following steps
+that will repeat every 24 hours following nightly observing and before release of the Solar System
+Daily Data Products (for more information on the overall Solar System Processing pipeline and Daily
+Data Products, see the :doc:`Solar System Processing pipeline </data-products-dp0-3/solar-system-processing-pipeline>` page):
+
+1. The nightly DIASource catalog is ingested.
+2. All stationary objects are removed from the nightly catalog of DIASources. In addition, any individual sources that can be attributed to known moving objects with high-confidence orbits where the astrometric match between the individual source and a known object has very low uncertainty (SNR>=5) are also removed from the nightly catalog of DIASources. In this manner, the moving object detections that are unattributed to any known sources are retrieved from the nightly DIASource catalog to be passed to candidate tracklet identification.
+3. All DIASources detected on the previous night that have not been matched at a high confidence level (SNR>=5) to a known Object, DIAObject, SSObject, or an artifact, are analyzed for potential pairs that form tracklets. These tracklets span 5 - 90 minutes and consist of a minimum of 2 detections, which can be mostly spurious and have unlimited overlap.
+4. Candidate tracklets are then passed to HelioLinC3D for linkage.
+5. HelioLinC3D is run to produce linkages that include tracklets from at least three distinct nights within a period of 14 days, with unlimited overlap allowed. Each linkage must comprise at least three tracklets – that is, at least six distinct detections of the candidate asteroid.
```

Comment:
Add a link to HelioLinC3D (either here, the previous line, or at the bottom of the page with other links -- or here _and_ at the bottom?).

Reply:
Yes, good idea!